packages/expo-camera/ios/EXCamera/EXCamera.m

```diff
@@ -509,7 +511,8 @@ - (void)record:(NSDictionary *)options resolve:(UMPromiseResolveBlock)resolve re
   }
 
   if (_movieFileOutput != nil && !_movieFileOutput.isRecording && _videoRecordedResolve == nil && _videoRecordedReject == nil) {
-
+    // Reset validation flag
+    _isValidRecordingConfig = YES;
```

Comment:
It might make more sense to place this inside of the `setVideoOptions` method (in the future, maybe we'll add a call to that method somewhere else, in which case we'd want to reset `_hasValidRecordingConfig` to `YES` then, too)